ndc-reference/bin/reference/main.rs

```diff
@@ -396,8 +428,59 @@ async fn get_schema() -> Json<models::SchemaResponse> {
         arguments: BTreeMap::new(),
     };
     // ANCHOR_END: schema_function_latest_article_id
+
+    // ANCHOR: schema_function_latest_article
+    let latest_article_function = models::FunctionInfo {
+        name: "latest_article".into(),
+        description: Some("Get the most recent article".into()),
+        result_type: models::Type::Nullable {
+            underlying_type: Box::new(models::Type::Named {
+                name: "article".into(),
+            }),
+        },
+        arguments: BTreeMap::new(),
+    };
+    // ANCHOR_END: schema_function_latest_article
+
+    // ANCHOR: schema_function_get_all_articles
+    let get_all_articles_function = models::FunctionInfo {
+        name: "get_all_articles".into(),
+        description: Some("Get all the articles".into()),
+        result_type: models::Type::Array {
+            element_type: Box::new(models::Type::Named {
+                name: "article".into(),
+            }),
+        },
+        arguments: BTreeMap::new(),
+    };
+    // ANCHOR_END: schema_function_get_all_articles
+
+    // ANCHOR: schema_function_get_article_by_id
+    let get_article_by_id_function = models::FunctionInfo {
+        name: "get_article_by_id".into(),
+        description: Some("Get article by ID".into()),
+        arguments: BTreeMap::from_iter([(
+            "id".into(),
+            models::ArgumentInfo {
+                description: Some("the id of the article to fetch".into()),
+                argument_type: models::Type::Named { name: "Int".into() },
+            },
+        )]),
+        result_type: models::Type::Nullable {
+            underlying_type: Box::new(models::Type::Named {
+                name: "article".into(),
+            }),
+        },
+    };
+    // ANCHOR_END: schema_function_get_article_by_id
+
     // ANCHOR: schema_functions
-    let functions: Vec<models::FunctionInfo> = vec![latest_article_id_function];
+    let functions: Vec<models::FunctionInfo> = vec![
```

Comment:
This bit gets included in the spec, and the same for `let procedures = ...`. We ought to include a change to the text to explain that these functions exist to illustrate various types of functions, and that we won't go into detail about each one in the text.

Reply:
Silly question, where do we update the text for these function explanations? ^^"